DOCUMENTATION.md

```diff
@@ -0,0 +1,271 @@
+
+- Adding a new remote repo
+
+```
+
+  
+
+![Picture4](https://user-images.githubusercontent.com/65342122/87232642-1f175400-c3de-11ea-8ead-80cd5ab4c37c.png)
+
+![enter image description here](https://user-images.githubusercontent.com/65342122/87232671-71587500-c3de-11ea-8ff5-25fb95b08553.png)
+
+  
+
+#### Add new branch
+
+  
+
+![Picture5](https://user-images.githubusercontent.com/65342122/87232643-23437180-c3de-11ea-9d9e-7e3c3789c32e.png)
+
+Note: In a newly initialized git repo, the newly added branch will be considered by git only after a initial commit
+
+  
+
+- Next, left pane consists of repositories, settings and help. In "repositories" choose a saved repo. This module has three sections : File view, Git difference and Git operation. The header portion shows chosen repo name, active branch, number of tracked files and commits.
+
+- File view shows the new/Untracked/Modified/deleted files.
+
+![Picture6](https://user-images.githubusercontent.com/65342122/87232644-29d1e900-c3de-11ea-9adc-03fb4e690882.png)
+
+- Git operations module consists of three basic git operations( Stage all changes, commit changes, push to remote ). Below this file status is display and the files can be staged with add action or click on "stage all changes" to stage all the untracked files from the chosen repo.
+
+  
+
+![Picture7](https://user-images.githubusercontent.com/65342122/87232645-30f8f700-c3de-11ea-8ddb-52f4d5ec7140.png)
+
+- After add operation all staged files can be removed immediately if you wish to, else ignore it. Reload the page to see the difference in "File View".
+
+![Picture8](https://user-images.githubusercontent.com/65342122/87232658-51c14c80-c3de-11ea-95e1-b9bbeeac82bb.png)
+
+- All staged files can be committed using "Commit changes" module. After commit changes reload the page to see active branch name, number of tracked files and number of commits.
+
+  
+
+![Picture9](https://user-images.githubusercontent.com/65342122/87232659-56860080-c3de-11ea-9bc4-a19ad727b101.png)
+
+- After **commit changes** operation,
+
+  
+
+![Picture10](https://user-images.githubusercontent.com/65342122/87232661-5c7be180-c3de-11ea-82b7-104792c2e3ec.png)
+
+  
+
+- "Push to remote" module pushes all commits to the remote host. If there is no remote in the chosen repo then ignore this module. After the push to remote is completed, check the remote host to see the pushed folder/file.
+
+with Remote(Git Hub)
+
+![Picture11](https://user-images.githubusercontent.com/65342122/87232662-61409580-c3de-11ea-8ad7-61c3871f0a4d.png)
+
+If you try to "push to remote" without having a remote host, "push failed" error message will be displayed.
+
+  
+
+![Picture12](https://user-images.githubusercontent.com/65342122/87232666-6867a380-c3de-11ea-9903-5ea12200e994.png)
+ 
+
+In "Git Difference" click on the modified file(s) to see the difference.
+
+![Picture13](https://user-images.githubusercontent.com/65342122/87232669-6d2c5780-c3de-11ea-9739-f8181e4d0901.png)
```

Comment:
Replace this image with a detailed code difference view. This image shows only a line change view